src/index.html

```diff
@@ -1,16 +1,50 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" id="document">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search--bar">
+      <form
+      class="search--bars bars--big"
+      data-qa="big"
+      >
+        <img src="../src/images/Search.svg" class="search__images images__big"
+        alt="search images">
+
+        <input
+        class="search__input input--big"
+        type="text"
+        placeholder="Try “Los Angeles”">
+      </form>
+
+
+      <form
+      class="search--bars bars--small "
+      data-qa="small"
+      >
+        <img src="../src/images/Search.svg" class="search__images images__small"
+           alt="search images">
+
+        <input
```

Comment:
There shouldn't be outline (remove it) on the focus
Also, the entire search bar should be clickable, not only this small part - i.e. input should be full-width and full-height

And there should be gradient on focus - see the mockup

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/f66b6074-9c75-40ba-b532-7f19c8fe1b57)

Reply:
i can add gradient only in input part

Reply:
fix code style here.
what about gradient - you can make the icon position absolute, add proper padding-left for input, and position your icon. (do not forget wrap your icon in label to make it also clickable)